core/types/block.go

```diff
@@ -109,6 +109,8 @@ type Header struct {
 	Verkle        bool
 	VerkleProof   []byte
 	VerkleKeyVals []verkle.KeyValuePair
+
+	DepositsRoot *libcommon.Hash // EIP-6110
```

Comment:
`DepositsRoot` should come before Verkle stuff because Verkle will happen after Pectra. Also, it should be changed to `RequestsRoot` with [EIP-7685](https://eips.ethereum.org/EIPS/eip-7685).